src/visual.ts

```diff
@@ -500,6 +507,23 @@ module powerbi.extensibility.visual {
                 .exit().remove();
 
             this.drawDots();
+            if (this.settings.axisoptions.show === true) {
+                this.SetAxisColor(this.settings.axisoptions.color);
+            } else {
+                this.SetAxisColor(LineDotChart.axesDefaultColor);
+            }
+        }
+
+        private SetAxisColor(color: string): void {
+            this.axisX.selectAll('line').style('stroke', function (d, i) { return color; });
```

Comment:
Can we color inherit? If so, please ally these settings to this.axisX and this.axisY